### PR TITLE
[feat] create admin sidenav, hide sidenav when user is not admin or o…

### PR DESCRIPTION
**Pull request by trevorhanus**
…wner

### CODE REVIEW

**Comment by RRDuke** [comment]
LGTM

**Comment by codydaig** [comment]
Missing Nav Controller Tests

File: modules/core/client/controllers/adminUsers.controller.client.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * @param {CustomService} Service that manages authentication
+ * @param {CustomService} Service that manages navigation
+ * @param {CustomService} Service that manages user sessions
+ */
+function AdminUsersController ($state, $mdSidenav, authService, navService, userService) {
+  var vm = this;
+  vm.state = $state;
+  vm.user = userService.currentUser;
+
+  //////////
```
Comment (codydaig) [inline modules/core/client/controllers/adminUsers.controller.client.js]: Unnecessary.

File: modules/core/client/services/user.service.client.js [hunk 2]
```diff
@@ -30,4 +31,12 @@ function userService() {
     user.currentUser = defaultUser;
     user.isLoggedIn = false;
   }
+
+  function hasRole (roles) {
+    if( Array.isArray(roles) ) {
+      return _.intersection(user.currentUser.roles, roles).length > 0;
+    }else{
```
Comment (codydaig) [inline modules/core/client/services/user.service.client.js]: Spaces

**Pull request merged**